src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -29,7 +34,8 @@ export async function deployImageToAzure(context: IActionContext, node?: RemoteT
         newSiteKind: AppKind.app
     };
     const promptSteps: AzureWizardPromptStep<IAppServiceWizardContext>[] = [];
-    const azureAccountTreeItem = await validateAzureAccountInstalled();
+    // Create a temporary azure account tree item since Azure might not be connected
```

Comment:
Connected as in the user isn't logged in yet?  Or is there special work to have Azure integrated with Docker (similar to GitHub?)

Reply:
Connected is referencing the "registries" view. You can deploy a docker hub image to azure, which means you may have never connected the Azure "registries" provider since your images are in Docker Hub, not Azure.